.github/workflows/add_contributor.yml

```diff
@@ -0,0 +1,51 @@
+name: Add Contributor to AUTHORS.md
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  add_contributor:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Check if contributor already exists
+        id: check_contributor
+        run: |
+          exists=$(grep -c -F "$GITHUB_ACTOR" AUTHORS.md)
+          if [[ $exists -gt 0 ]]; then
+            echo "::set-output name=exists::true"
```

Comment:
github has deprecated `set-output` for actions - please [see here](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) for more info & examples of how to refactor code to align with their new standards. would you be able to make similar updates to your code here?